examples/tutorial_server_dataaccess.c

```diff
@@ -0,0 +1,91 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information. */
+/**
+ * Integrating a DataAccess example VarieableType with Properties
+ * DataAccess defines new Variable Types and Properties for describing real time data.
+ * In this example a VariableType node is added with its respective DataAccess Properties
+*/
+#include "open62541.h"
+#include <signal.h>
+#include "ua_server_dataaccess.h"
+
+UA_Boolean running = true;
```

Comment:
static UA_Boolean running = true
to limit the scope of the symbol to this translation unit (just like it is already done for the functions)?